website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -316,6 +316,11 @@ While much of dbt's adapter-specific functionality can be modified in adapter ma
 
 </File>
 
+### Enabling dbt init 
+
+In order to enable the [dbt init command](/reference/commands/init), make sure to include a sample profile file. The filepath will be `dbt/adapters/<adapter_name>/sample_profiles.yml`. This will assure that users can create a new dbt project with the command. 
+For sample profiles, check out this [example](https://github.com/dbt-labs/dbt/blob/develop/plugins/postgres/dbt/include/postgres/sample_profiles.yml).
+
```

Comment:
I'd love to expand this to `### Other files`, and mention:
- `dbt/include/<adapter_name>/sample_profiles.yml` (for the exact reason you say)
- `dbt/adapters/<adapter_name>/__version__.yml` (for the output of `dbt --version`)

FWIW both of those files are now included in the bootstrapped output of `create_adapter_plugins.py` (https://github.com/dbt-labs/dbt/pull/3509)

Reply:
Ah happy to do this. What does it mean that it's included in the bootstrapped output?

Reply:
When someone uses the `create_adapter_plugins.py` script, the output will now include `sample_profiles.yml` and `__version__.py`